app.json

```diff
@@ -3,12 +3,6 @@
   "scripts": {
   },
   "env": {
-    "ADEPT_SCALE_LICENSE_KEY": {
-      "required": true
-    },
-    "ADEPT_SCALE_URL": {
-      "required": true
-    },
     "APPLICATION_URI": {
```

Comment:
this probably needs to be the review app application uri (i don't know if we rely on this for cors, etc.) but I don't know how to set that here. I think for now maybe we can just leave this as you have it.